core/keyserver/validate.go

```diff
@@ -44,12 +44,12 @@ var (
 	ErrNoCommitted = errors.New("missing commitment")
 	// ErrCommittedKeyLen occurs when the committed key is too small.
 	ErrCommittedKeyLen = errors.New("committed.key is too small")
-	// ErrWrongIndex occurs when the index in key value does not match the
-	// output of VRF.
-	ErrWrongIndex = errors.New("index does not match VRF")
```

Comment:
Can you please keep this here? This will make the diff concise. Unless there's a reason for moving it.